tools/swarm/src/cli.rs

```diff
@@ -27,7 +27,7 @@ pub struct Cli {
     pub no_banner: bool,
     /// Path to a directory with Iroha configuration. It will be mapped as volume for containers.
     ///
-    /// The directory should contain `config.json` with `genesis.file` parameter specified.
+    /// The directory should contain `config.json` and `genesis.json`
```

Comment:
Yes, but soon enough it will be `toml` without multi-extension support. And it is also possible to put JSON5 with `.json` extension and Iroha will be fine with that.